docker.sh

```diff
@@ -205,10 +205,10 @@ check_if_tag_exists(){
 
 check_local_remote_are_even(){
     REMOTE_COMMIT=$(git ls-remote $GITBOX_URL | head -1 | sed "s/[[:space:]]HEAD//")
-    LOCAL_COMMIT=$(git rev-parse master)
+    LOCAL_COMMIT=$(git rev-parse main)
 
     if [[ $REMOTE_COMMIT != $LOCAL_COMMIT ]]; then
-        echo "Master in Apache repository is not even with local master"
+        echo "Master in Apache repository is not even with local main"
```

Comment:
I think `Master` should also be replaced here :)

Reply:
Good point. I was just searching for master lowercase so I missed this. Fixed.